Sources/ManageServer.php

```diff
@@ -302,7 +302,7 @@ function AlignURLsWithSSLSetting($new_force_ssl = 0)
 	$request = $smcFunc['db_query']('', '
 		SELECT id_theme, variable, value
 		  FROM {db_prefix}themes
-		 WHERE variable in ({string:themeurl}, {string:imagesurl})
+		WHERE variable in ({string:themeurl}, {string:imagesurl})
```

Comment:
There's some weird indentation happening in this whole query. It should probably all be fixed, not just this one line.

Reply:
What would you say we should use a default indentation for queries? I don't normally deal with them and I can't quite get something in my head between the inconsistencies and the different format some editors suggest me. Though I see we always indent AND.

Reply:
As a general rule, we align the SQL keywords that start a main logical clause of a query (e.g. SELECT, FROM, WHERE, UPDATE, ORDER BY, etc.) each on their own line and using the same indentation. 

The main exception is the JOIN keywords (JOIN, INNER JOIN, LEFT JOIN, etc.) which are always indented one more tab stop. This is because JOIN ultimately functions to expand the number of tables used in the FROM clause, and so we indent it in a way that shows it as subordinate to FROM.

Keywords that function more like inline logic within a clause (e.g. AND, OR, IN, AS, ASC, DESC, etc.) can either be left inline or moved to a new line that indented another tab stop to show that it is subordinate to the clause it is part of.

---

The reason that this particular case is weird is simply that someone decided to right-align the major keywords (SELECT, FROM, WHERE) instead of left-aligning them. Just take out the extra spaces in front of those words, and this one should be good.

Reply:
Alright. I'll do a second revision for those and the breaks then.